airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -943,6 +962,14 @@ def run_load(self,
                 'timePartitioning': time_partitioning
             })
 
+        if cluster_fields:
```

Comment:
Maybe separate this logic to a separate function? Since it is defined twice.

Reply:
Agree with @Fokko

Reply:
After rebasing this morning, the changes made by AIRFLOW-491 mean the logic for `run_query` and `run_load` have diverged somewhat, so factoring it out probably now makes less sense.